mrmustard/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Xanadu Quantum Technologies Inc.
+# Copyright 2023 Xanadu Quantum Technologies Inc.
```

Comment:
I'm not sure we want to advance the dates of the original copyright. Let's ask David.